<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QDialog" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="verify_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="deposit_label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>存款金额：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="damount_edit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>verify_button</sender>
   <signal>clicked()</signal>
   <receiver>deposit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
